docs/source/starters/new_project_tools.md

```diff
@@ -1,4 +1,250 @@
+
+```bash
+pip install -r path/to/project/root/requirements.txt
+```
+
+The linting tool will configure `ruff` with the following settings by default:
+```toml
+#pyproject.toml
+
+[tool.ruff]
+line-length = 88
+show-fixes = true
+select = [
+    "F",   # Pyflakes
+    "W",   # pycodestyle
+    "E",   # pycodestyle
+    "I",   # isort
+    "UP",  # pyupgrade
+    "PL",  # Pylint
+    "T201", # Print Statement
+]
+ignore = ["E501"]  # Black takes care of line-too-long
+```
+
+With these installed, you can then make use of the following commands to format and lint your code:
+
+```bash
+ruff format path/to/project/root
+black path/to/project/root --check
+```
+
+Though it has no impact on how your code works, linting is important for code quality because improves consistency, readability, debugging, and maintainability. To learn more about linting your Kedro projects, check our [linting documentation](../development/linting.md).
+
+### Testing
+
+This tool introduces the `tests` directory to the new project's structure, containing the file `test_run.py` with an example unit test. [`Pytest`](https://docs.pytest.org/en/7.4.x/) is added as a dependency in your new project's requirements. After project creation, make sure it is installed by running the following command:
+
+```bash
+pip install -r path/to/project/root/requirements.txt
+```
+
+| ![cli-output.jpg](IMAGE RESOURCE PLACEHOLDER) |
+|:--:|
+| *Prompt for tools selection* |
+
+The tool leverages `pytest` with the following configuration:
+
+```toml
+[tool.pytest.ini_options]
+addopts = """
+--cov-report term-missing \
+--cov src/{{ cookiecutter.python_package }} -ra"""
+
+[tool.coverage.report]
+fail_under = 0
+show_missing = true
+exclude_lines = ["pragma: no cover", "raise NotImplementedError"]
+```
+
+To run your tests, use the following command:
+```bash
+pytest path/to/your/project/root/tests
+```
+
+Kedro promotes the use of unit tests to achieve high code quality and maintainability in your projects. To read more about unit testing with Kedro, check our [testing documentation](../development/automated_testing.md#set-up-automated-testing-with-pytest)
+
+### Custom logging
+
+Selecting the custom logging tool introduces the file `logging.yml` to your project's `conf` directory.
+
+| ![cli-output.jpg](IMAGE RESOURCE PLACEHOLDER) |
+|:--:|
+| *Prompt for tools selection* |
+
+This tool allows you to customise your logging configuration instead of using [Kedro's default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml). The populated `conf/logging.yml` provides two additional logging handlers: `console` and `info_file_handler`, as well as `rich` that is available in the default configuration, though only `info_file_handler` and `rich` are used.
+
+To learn more about using logging in your project, or modifying the logging configuration, take a look at our [logging documentation](../logging/index.md).
+
+### Documentation
```

Comment:
This is subject to change depending on: https://github.com/kedro-org/kedro/issues/3376